src/components/radiobutton/radio-button-group/radio-button-group.directive.ts

```diff
@@ -0,0 +1,91 @@
+import { AfterContentInit, ContentChildren, Directive, EventEmitter, ExistingProvider, forwardRef, Input, OnDestroy, Output, QueryList } from '@angular/core';
+import { RadioButtonComponent } from '../radiobutton.component';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { takeUntil } from 'rxjs/operators';
+import { Subject } from 'rxjs';
+
+/**
+ * Needed for Reactive forms so there is a top level element
+ */
+
+export const RADIO_GROUP_CONTROL_VALUE_ACCESSOR: ExistingProvider = {
+    provide: NG_VALUE_ACCESSOR,
+    useExisting: forwardRef(() => RadioButtonGroupDirective),
+    multi: true
+};
+
+@Directive({
+    selector: 'ux-radio-button-group, [uxRadioButtonGroup]',
+    providers: [
+        RADIO_GROUP_CONTROL_VALUE_ACCESSOR
+    ],
+    host: {
+        'role': 'radiogroup'
```

Comment:
With this in place, we no longer need to have `role="radiogroup"` in the example HTML.

Reply:
removed from example and snippet code